docs/DEPLOYMENT.md

```diff
@@ -0,0 +1,132 @@
+# Development
+
+## Setup Tools
+
+### CRC
```

Comment:
could you please avoid `crc` `oc` and use `kubectl`
and also test in on gcp or kind..

Reply:
Works fine on Minikube